src/vending-machine.js

```diff
@@ -0,0 +1,61 @@
+const processData = require('./../__mock__/process-data');
+
+class vendingMachine {
+    constructor(data) {
+        this.data = processData(data);
+        // console.log(this.data)
+    }
+
+    printInvintory () {
+        return this.data.invintory;
+    }
+
+    refillInvintory () {
+        const refill = 30;
+        for( var i = 0; i < this.data.invintory.length; i++ ) {
```

Comment:
We used `for` loops primarily when we were doing optimizations. In this use-case, a `forEach` loop would be far simple and easier to read.